### ISSUE TITLE
Environment Parameters do not activate/deactivate in Powershell

### ISSUE THREAD

**Issue by phreed** [mamba-org/mamba#2644]
I look in https://github.com/mamba-org/mamba/blob/16c3da8e4dc2228f5dbe7e9b4b5d07812b9a55c2/libmamba/src/core/activation.cpp#L979-L988 and I see bat and sh but no ps1.

In conda the environment variables activate/deactivate in a different manner.
Setting environment variables in cmd.exe and bash are described here. 
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#saving-environment-variables

The installation instructions for `minimamba` has instructions for powershell, https://mamba.readthedocs.io/en/latest/installation.html#windows
This is inconsistent.

Cause  `env_var.ps1` files to be supported in activate/deactivate.

**Comment by AntoinePrv** [comment]
Hi @phreed, I am not particularly knowledgeable about Windows, would you mind giving an example of that you observe vs what you expect?

You mention you have an issue in Powershell, but point to code for `cmd.exe` activation. There is a different `PowerShellActivator`. Which one are you working with?

>  This is inconsistent.

Could you detail what behavior is inconsistent with what other behavior?

> Cause env_var.ps1 files to be supported in activate/deactivate.

Do you mean you write a `env_var.ps1` in `%CONDA_PREFIX%\etc\conda\activate.d\` that is not red when activating?

**Comment by phreed** [comment]
> Hi @phreed, I am not particularly knowledgeable about Windows, would you mind giving an example of that you observe vs what you expect?
> 
> You mention you have an issue in Powershell, but point to code for `cmd.exe` activation. There is a different `PowerShellActivator`. Which one are you working with?
> 
> > This is inconsistent.
> 
> Could you detail what behavior is inconsistent with what other behavior?
> 
> > Cause env_var.ps1 files to be supported in activate/deactivate.
> 
> Do you mean you write a `env_var.ps1` in `%CONDA_PREFIX%\etc\conda\activate.d\` that is not read when activating?

Yes, that is what I meant.
I see that I was wrong (not reading the code correctly).

**Comment by phreed** [comment]
Now that I have looked at the code I find that the following works.
1. Create a directory `~/micromamba/envs/<env-name>/etc/conda/env_vars.d/`.
2. Put a json file in that directory. `~/micromamba/envs/<env-name>/etc/conda/env_vars.d/<foo>.json`.
3. Define some variables it that file.

```json
{
  "FOO": "bar"
}
```
This works on Windows (and I presume the others). However, I do not see this in the documentation.

**Comment by phreed** [comment]
> Do you mean you write a env_var.ps1 in %CONDA_PREFIX%\etc\conda\activate.d\ that is not read when activating?

I did a test, putting the following in ```$CONDA_PREFIX/etc/conda/activate.d/env-var.ps1.
```powershell
$Env:PROJECT_DIR = "C:\Users\me\Desktop\cts-3.x"

New-Item "C:\Users\me\foo.txt"
Set-Content "C:\Users\me\foo.txt" 'the env_var.ps1 ran'
```
When I activate the corresponding conda environment the `foo.txt` file is created but that environment does not have the $Env:PROJECT_DIR .
So, the activation scripts, apparently, should not be used for setting environment parameters (at least in powershell).

**Comment by AntoinePrv** [comment]
> When I activate the corresponding conda environment the foo.txt file is created but that environment does not have the $Env:PROJECT_DIR .
So, the activation scripts, apparently, should not be used for setting environment parameters (at least in powershell).

I don't know enough about powershell to tell whether if this is a bug or not possible (script running in a subshell).

> This works on Windows (and I presume the others). However, I do not see this in the documentation.

This should be documented, however, right now we do not have detailed doc on all features and been relying a lot on Conda.
However I don't see it documented there...

**Comment by jjerphan** [comment]
Duplicate of #1717 which has been fixed in 2.0.
